app/models/manageiq/providers/redhat/infra_manager/ems_cluster.rb

```diff
@@ -0,0 +1,29 @@
+class ManageIQ::Providers::Redhat::InfraManager::EmsCluster < ::EmsCluster
```

Comment:
@borod108 lets try to define this as

module ManageIQ
  module Providers
    module Redhat
      class InfraManager
         class EmsCluster
      end
    end
  end
end